paasta_itests/steps/setup_marathon_job_steps.py

```diff
@@ -58,14 +52,53 @@ def create_complete_app(context):
             pass
 
 
+@when(u'we set up an app to use zookeeper scaling with {number:d} max instances')
+def setup_zookeeper(context, number):
+    client = KazooClient(hosts='%s/mesos-testcluster' % get_service_connection_string('zookeeper'), read_only=True)
+    client.start()
+    try:
+        client.delete('/autoscaling', recursive=True)
```

Comment:
I take it this is here to cleanup between runs? If so, can you move this to ``environment.py`` where we have the other cleanup code (like removing runnning marathon apps, chronos jobs)